Apps/Sandcastle/gallery/Materials.html

```diff
@@ -202,6 +202,29 @@
      });
 }
 
+function applyCompressedTextureMaterial(primitive, scene) {
+    Sandcastle.declare(applyCompressedTextureMaterial); // For highlighting in Sandcastle.
+
+    var compressedImageUrl;
+    var context = scene.context;
```

Comment:
`Context` is not part of the public API.  You could expose these on `Scene` or perhaps a `scene.getTextureFormatSupported(String)` function.  Whatever you think is clean.